CIRC-10032 - httptrap Documentation Improvements (#480)
circonus/docs: content/circonus/integrations/library/json-push-httptrap.md

 * `{ "_type": "L", "_value": "2187345234234" }` becomes the unsigned, 64-bit
   integer `2187345234234`.

In addition, you may submit a flags field, `_fl`, to change the data storage behavior. The default behavior, if no

> flags are specified, is to always replace the existing value with the latest one, but you can change that behavior using

Reviewer: ```suggestion
flags are specified, is to always replace any unflushed value in the current period with the latest one, but you can change that behavior using
```

the `_fl` field. The accepted values are:

> * `+` - Accumulate all values
> * `~` - Average all values

Reviewer: ```suggestion
* `+` - Sum all sample values for this metric in the collection period
* `~` - Average all sample values for this metric in the collection period
```

For example, if the following values are submitted:

```

>   { "_type": "L", "_value": "1", "_fl": "+" }
>   { "_type": "L", "_value": "1", "_fl": "+" }
> ```
> 
> ...the emitted value will be `2`. If the following are submitted:

Reviewer: ```suggestion
  { "_type": "L", "_value": "10", "_fl": "+" }
  { "_type": "L", "_value": "20", "_fl": "+" }
```

...the emitted value will be `30`. If the following are submitted:
```
